gui/wxpython/core/toolboxes.py

```diff
@@ -757,9 +757,9 @@ def new_translator(string):
     sys.displayhook = new_displayhook
     sys.__displayhook__ = new_displayhook
 
-    import __builtin__
+    import builtins as __builtin__
 
-    __builtin__._ = new_translator
+    __builtin__.__dict__["_"] = new_translator
```

Comment:
You are editing the line with usage anyway, so let's change `__builtin__` to `builtins`.

Reply:
Done